.rubocop_todo.yml

```diff
@@ -1,3 +1,5 @@
+require: rubocop-rspec
```

Comment:
Nope. They're separate files. `.rubocop.yml` is where we set the styles we care about, and `.rubocop_todo.yml` is where we put temporary exceptions to appease the :cop:. I started off by having this `require` statement in the former file, but that causes complaints because the todo file makes a reference to rspec-related cops. I *could* put the `require` statement in both files, but that seems janky. (I don't love having the `require` in the TODO file which will eventually go away. I'm open to being told that having the `require` in both files is less janky, though.)